scripts/superpmi.py

```diff
@@ -407,7 +410,10 @@ def collect(self):
                     self.__collect_mc_files__()
                 
                 if not self.coreclr_args.has_merged_mch:
-                    self.__merge_mc_files__()
+                    if not self.coreclr_args.merge_mch_files:
+                        self.__merge_mc_files__()
```

Comment:
It seems to me like we should always merge whatever we need to, either mc or mch. I worry about adding this logic to distinguish between them.

Ideally, mcs `-merge` would subsume `-concat` and do any and all merging required (although the usage model would be a little more confusing if you also want to use `-recursive`).

Reply:
I agree, I think this should be added to the backlog. Until then for expedience I will rely on the current functionality.